src/Provider/Provider.Web/Views/Part2/ApplicationProcess/ApplicationProcess.cshtml

```diff
@@ -42,13 +42,16 @@
                         </div>
                         <div id="application-method-external-panel" class="govuk-radios__conditional govuk-radios__conditional--hidden">
                             <div esfa-validation-marker-for="ApplicationUrl" class="govuk-form-group">
-                                <label asp-for="ApplicationUrl" class="govuk-label">Enter a link about the vacancy. The page should give applicants a way to apply</label>
+                                <label asp-for="ApplicationUrl" class="govuk-label">Application website link</label>
                                 <span esfa-validation-message-for="ApplicationUrl" class="govuk-error-message"></span>
                                 <input asp-for="ApplicationUrl" class="govuk-input govuk-!-width-two-thirds" maxlength="2000" />
```

Comment:
This also needs changing along with the validation change s0 that `data-maxlength` matches